ci/generate.nix

```
@@ -4,6 +4,7 @@
   efmls-configs-sources,
   none-ls-builtins,
   lspconfig-servers,
+  conform-formatters,
   nixfmt-rfc-style,
```

```suggestion
  nixfmt,
```

nit: unrelated to your change, but this should point to stable now